lighthouse-core/config/constants.js

```diff
@@ -92,6 +93,7 @@ const defaultPassConfig = {
 };
 
 const nonSimulatedPassConfigOverrides = {
+  pauseAfterFcpMs: 5250,
   pauseAfterLoadMs: 5250,
   networkQuietThresholdMs: 5250,
   cpuQuietThresholdMs: 5250,
```

Comment:
maybe we could have gotten away with a single `pauseAfterLoadingLandmark` that just waits that much after the last of these, but there's probably unavoidable trickery in there, and that ship sailed a long time ago regardless